patches/chrome-app-BUILD.gn.patch

```diff
@@ -1,22 +1,21 @@
 diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
-index 8179dd92317a4cbafb65bdefd5c27f9848bca0d7..411ad5c908ceb4aac150a5ac1844d288057c71f2 100644
+index c8d3aa8b054a285b6340f60d1e4fffe854070ae7..9aa372bd954df217e758e732542edf0e780bf16e 100644
 --- a/chrome/app/BUILD.gn
 +++ b/chrome/app/BUILD.gn
-@@ -124,6 +124,8 @@ grit("generated_resources") {
+@@ -117,6 +117,7 @@ grit("generated_resources") {
    if (is_android) {
      outputs += android_generated_java_resources
    }
-+
 +  deps = [ "//brave/app:brave_generated_resources_grit" ]
  }
  
  if (is_android) {
-@@ -159,7 +161,7 @@ grit("google_chrome_strings") {
- }
+@@ -139,7 +140,7 @@ grit("chromium_strings") {
+   outputs = [ "grit/chromium_strings.h" ] +
+             process_file_template(
+                 all_chrome_locales,
+-                [ "${branding_path_product}_strings_{{source_name_part}}.pak" ])
++                [ "${branding_path_component}_strings_{{source_name_part}}.pak" ])
```

Comment:
this seems wrong, we were overriding this because it had a fixed value of chromium before, but now we shouldn't need this patch

Reply:
looks like these were fixed later